packages/shared/src/components/utilities/common.tsx

```diff
@@ -164,6 +164,10 @@ export type HTMLElementComponent<T = HTMLElement> = React.FC<HTMLAttributes<T>>;
 export enum SharedFeedPage {
   MyFeed = 'my-feed',
   Popular = 'popular',
+  Explore = 'explore',
+  ExploreLatest = 'explorelatest',
+  ExploreDiscussed = 'explorediscussed',
+  ExploreUpvoted = 'exploreupvoted',
```

Comment:
Not a fan, but due to the way we store feed names, we don't allow special characters.

Reply:
What about like`my-feed` in kebab case?

Reply:
Each page is under the explore directory. Feels like it is more appropriate instead of making each page under the root with hyphens, but that is just my opinion. I could be wrong though.

Reply:
nah no, it's better the way u've done it